include/OpenSpaceToolkit/Astrodynamics/EventCondition.hpp

```diff
@@ -0,0 +1,122 @@
+/// Apache License 2.0
+
+#ifndef __OpenSpaceToolkit_Astrodynamics_EventCondition__
+#define __OpenSpaceToolkit_Astrodynamics_EventCondition__
+
+#include <OpenSpaceToolkit/Core/Types/Real.hpp>
+#include <OpenSpaceToolkit/Core/Types/String.hpp>
+
+#include <OpenSpaceToolkit/Mathematics/Objects/Vector.hpp>
+
+namespace ostk
+{
+namespace astro
+{
```

Comment:
I think it makes sense yeah. But I think Vishwa added them at the root for now (similar to NumericalSolver) because will move it to math (hopefully soon)

Reply:
Oohhh yeah that makes sense to me. Vishwa, if you confirm feel free to close this comment

Reply:
Yeah precisely, there is a namespace for the different conditions but not for the parent object. Once we move it to Math we can figure what we want to do with it.